dashboards/gateway-api-inference-extension/inference-extension-prometheus.json

```diff
@@ -0,0 +1,763 @@
+                    "unitOverride": ""
+                  }
+                },
+                {
+                  "breakdowns": [],
+                  "dimensions": [],
+                  "legendTemplate": "90%",
+                  "measures": [],
+                  "plotType": "LINE",
+                  "targetAxis": "Y1",
+                  "timeSeriesQuery": {
+                    "outputFullDuration": false,
+                    "prometheusQuery": "histogram_quantile(0.9, sum by(le) (rate(inference_model_response_sizes_bucket{}[${__interval}])))",
+                    "unitOverride": ""
+                  }
+                },
+                {
+                  "breakdowns": [],
+                  "dimensions": [],
+                  "legendTemplate": "50%",
+                  "measures": [],
+                  "plotType": "LINE",
+                  "targetAxis": "Y1",
+                  "timeSeriesQuery": {
+                    "outputFullDuration": false,
+                    "prometheusQuery": "histogram_quantile(0.5, sum by(le) (rate(inference_model_response_sizes_bucket{}[${__interval}])))",
+                    "unitOverride": ""
+                  }
+                }
+              ],
+              "thresholds": []
+            }
+          }
+        },
+        {
+          "yPos": 57,
+          "height": 16,
+          "width": 19,
+          "widget": {
+            "title": "Input Token Count",
+            "id": "",
+            "xyChart": {
+              "chartOptions": {
+                "displayHorizontal": false,
+                "mode": "COLOR",
+                "showLegend": false
+              },
+              "dataSets": [
+                {
+                  "breakdowns": [],
+                  "dimensions": [],
+                  "legendTemplate": "95%",
+                  "measures": [],
+                  "plotType": "LINE",
+                  "targetAxis": "Y1",
+                  "timeSeriesQuery": {
+                    "outputFullDuration": false,
+                    "prometheusQuery": "histogram_quantile(0.95, sum by(le) (rate(inference_model_input_tokens_bucket{}[${__interval}])))",
+                    "unitOverride": ""
+                  }
+                },
+                {
+                  "breakdowns": [],
+                  "dimensions": [],
+                  "legendTemplate": "90%",
+                  "measures": [],
+                  "plotType": "LINE",
+                  "targetAxis": "Y1",
+                  "timeSeriesQuery": {
+                    "outputFullDuration": false,
+                    "prometheusQuery": "histogram_quantile(0.9, sum by(le) (rate(inference_model_input_tokens_bucket{}[${__interval}])))",
+                    "unitOverride": ""
+                  }
+                },
+                {
+                  "breakdowns": [],
+                  "dimensions": [],
+                  "legendTemplate": "50%",
```

Comment:
With all the other GKE AI dashboards (vLLM, Triton, etc) we used `p50`, `p95`, `p99` instead of `50%`, `90%`, `95%`.

Reply:
Done.